### ISSUE TITLE
Unable to sign in on dev branch

### ISSUE THREAD

**Issue by sephallen** [ruippeixotog/docker-google-musicmanager#4]
I am using the dev branch because my os does not support ipv6 and it has been running great.
Recently however, I am no longer able to sign into the music manager.
After inputting my credentials (I'm using two factor authentication) which I am prompted for, another message appears saying "Couldn't connect to Google Play. Check your internet connection and try again".
My connection is fine, and the fact that I'm prompted to enter my two factor code leads me to believe that the docker is also receiving a connection fine.

**Comment by ruippeixotog** [comment]
I tried to reproduce your issue @sephallen, but I wasn't able to. I used the following command:

```
sudo docker run -dt --net=host -v /local/music/dir:/music -v /local/appdata/dir:/appdata -e SERVER_NAME=server-name -p 5900:5900 ruippeixotog/google-musicmanager:dev
```

It prompted me for the (two factor) authentication and then everything worked as expected. Is it possible that anything changed recently in the network configuration of your system? Can you try starting a fresh new container (i.e. not mounting or mounting empty folders for music and appdata) and see if the problem persists?

**Comment by sephallen** [comment]
Thank you for the reply, I have tried everything you suggested, but unfortunately the problem still persists.

**Comment by ruippeixotog** [comment]
I just updated the dev branch with the latest changes from master, particularly the addition of a mac address spoofer so that the `--net=host` option is not needed anymore. Can you pull the new `dev` Docker image and try running it again without `--net=host`?

**Comment by sephallen** [comment]
OK so I seem to be getting further, but now I get the message:

"Only two accounts can be used per computer"

But I've never used Google Music with any other account ever.

**Comment by ruippeixotog** [comment]
That's very strange. Can it be related to [these reports](https://productforums.google.com/forum/#!topic/apps/LrdGrLt8C2U) of the same problem?

**Comment by sephallen** [comment]
Yeah probably the same, but with your new mac spoofer, this shouldn't be a problem?

**Comment by JasonGhent** [comment]
I'm running into the same issue of `Only two accounts can be used per computer`. Perhaps using `--net=host` would resolve this? Due to hardware limitations, I am unable to test it and find out.

**Comment by ruippeixotog** [comment]
It seems the mac spoofer is having some problems. I cannot test it right now, but if you are able to run it outside a VM, using the image without mac spoofing (`-e SPOOF_MAC=false`) and running with the `--net=host` option should work.

**Comment by stevephillips** [comment]
Hi, Just installed this docker and I'm also getting the "Only two accounts can be used per computer" message. Can anyone confirm a fix? I don't want to fiddle around with commands and potentially add authorised devices to my account until a definite fix is done.

**Comment by TwanoO67** [comment]
Hello!
Same problem here.
I tried with the -e SPOOF_MAC=false and --net=host
this doesn't solve the problem, and now the vnc server has a black screen.
I don't know what to test next...

**Comment by ironlion27** [comment]
`-e SPOOF_MAC=false`
&
`--net=host`

Worked for me.  Thanks!

**Comment by ironlion27** [comment]
Having the same issue @TwanoO67 is having.  If I go just `-e SPOOF_MAC=false` I get the error 

> Could not identify your computer. 

via VNC interface.

**Comment by ironlion27** [comment]
Upon more research I came across this group forum [https://productforums.google.com/d/msg/play/L-8DRNwVF_o/1nW0B9itw-gJ](url) that leads me to believe if there was a way to change the docker containers MAC Address we could resolve this.

I am not sure how to run commands on a container but if someone who does can advise on how to run macchanger on this container to see if the issue resolves.  It would be greatly appreciated.

**Comment by fun4stuff** [comment]
Anybody figure out how to fix this bug, "Only two accounts can be used on this computer" ? Or is this project DOA?

**Comment by ruippeixotog** [comment]
The only thing I can say is that the image still works for me. The last command I used to launch the image (which, granted, was almost a year ago) was:

```
docker run -d --restart=always --name=gmm -t --net=host -p 5900:5900 \
  -v <music_dir>:/music -v <app_dir>:/appdata \
  -e SERVER_NAME=<name> -e SPOOF_MAC=false \
  ruippeixotog/google-musicmanager
```

I'm running this on a Ubuntu 16.04.2 LTS barebones machine.

Unfortunately, I really am unable to debug this. The official Google Music Manager application has no native support for headless servers and, therefore, has little to no documentation regarding this use case. I built this image following the scarce instructions I found in the internet for running this by emulating an X environment, but I don't have sufficient knowledge on how to avoid the checks Google seems to do.

Looking at the problems this image is having, I believe that an image wrapping a project such as https://github.com/simon-weber/gmusicapi would be more stable, even though it is an unofficial solution.
